File: Python/Ex34.py
numeros = []
for i in range(0, 10, 1):
    x = int(input('Digite um numero: '))
    numeros.append(x)
print('Os numeros digitados em ordem crescente foram: ')    
for i in range(0, 10, 1):
    print(numeros[i])
for i in range(0, 10, 1):
  if numeros[0]<numeros[i]:
     numeros[0]=numeros[i]

print("O maior número digitado foi:", numeros[0])

File: C/Ex34.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){
    int h, numeros[10];

    printf("Digite 10 numeros: ");
    for(int i=0; i<10; i++){
        scanf("%i", &numeros[i]);
        }
    printf("Os numeros digitados foram: ");
  for(int i=0; i<10; i++){
      printf("%i \n", numeros[i]);
  }
    h=numeros[0];
    for(int i=1; i<10; i++) {
        if(numeros[i] > h){
            h=numeros[i];
            }
   }
     printf("O maior numero foi: %i ", h);
   return 0;
   }         

